api/src/test/java/ca/bc/gov/educ/studentdatacollection/api/rules/RulesProcessorTest.java

```diff
@@ -492,7 +492,9 @@ void testGradeCodeRule() {
         entity.setEnrolledGradeCode("X");
         val validationErrorCode = rulesProcessor.processRules(createMockStudentSagaData(SdcSchoolCollectionStudentMapper.mapper.toSdcSchoolStudent(entity), createMockSchool()));
         assertThat(validationErrorCode.size()).isNotZero();
-        assertThat(validationErrorCode.get(0).getValidationIssueCode()).isEqualTo("INVALIDGRADECODE");
+        System.out.println(validationErrorCode);
```

Comment:
Is this println wanted, or was it for debugging?

Reply:
Oooo- debugging. I'll remove it